README.md

```diff
@@ -25,7 +25,7 @@ If you are using Windows and python was installed using the [python installer](h
 
```

Comment:
If that option was not used and python was manually added to the path this

needs a comma "added to the path, this"

Reply:
done